### ISSUE TITLE
A few problems with link validation

### ISSUE THREAD

**Issue by TheThirdOne** [hackclub/markcop#8]
`htts://jsbin.com` is a valid link.

I am not sure if `workshop_details#general-workshop-facilitation-guidelines` is valid, but it should at least give a better error.

Not sure if this is a problem in markcop or the pr, but it should be looked at.

```
cat: /home/ubuntu/hackedu/playbook/workshops/workshop_details: No such file or directory
```

Full logs can be found [here](https://circleci.com/gh/hackedu/hackedu/359).

**Comment by maxwofford** [comment]
@TheThirdOne I'm going to keep the error for `htts://jsbin.com` being an invalid link because it is missing `https`. Would `$LINK is not a valid link` be a better error?

For the `cat` issue, @zachlatta said it should be fixed by correcting all the links in the PR. 

The new markcop will be rolled out sometime today, so let's wait and see if those errors still occur.

**Comment by TheThirdOne** [comment]
@MaxWofford, I misread `htts://jsbin.com`.

And sounds good.

**Comment by maxwofford** [comment]
These fixes were merged by #9
